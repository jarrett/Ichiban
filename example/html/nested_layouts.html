<% layouts 'nested_outer', 'nested_inner' %>

<p>This is the content for the nested layouts.</p>