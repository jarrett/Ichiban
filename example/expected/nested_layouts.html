<!DOCTYPE html>
<html>
  <head>
    <title>Example Site</title>
  </head>
  
  <body>
    <h1>Nested Layout - Outer</h1>
    
    <div id="inner">
  <h2>Nested Layout - Inner</h2>
  
  
<p>This is the content for the nested layouts.</p>
</div>
  </body>
</html>